### PR TITLE
Added downloading of remote datasets to pipeline

### LINKED ISSUE #19: Download dataset as part of pipeline
The dataset is stored outside the project and can be automatically downloaded as part of the pipeline. The dataset is stored remotely and the feature encoding is a shared component.

### PR DESCRIPTION
**Pull request by SagaRut**
Since dvc pull is having issues then I added this to the pipeline just in case. We may want to refactor things around it though, but for now this is a quick fix to hand in tonight.

### CODE REVIEW

File: src/data/get_data.py [hunk 1]
```diff
@@ -18,6 +19,9 @@ def main():
     test_data = sys.argv[2]
     val_data = sys.argv[3]
 
+    gdown.download_folder('https://drive.google.com/drive/folders/1_NobSEMZS8jogSAEZ9ZLBUTemPiASJRg',
+                          output="data", quiet=False)
```
Comment (blibliboe) [inline src/data/get_data.py:23]: I don't know if this is the correct solution as they are already downloaded with the dvc pull. And with a cleared cache it worked for everyone.

Comment (SagaRut) [reply src/data/get_data.py:23]: Now it is a part of the pipeline though and not just pulled. Also it didn't work for some by just clearing the cache, fetching each individual file was needed which is a hassle for people reviewing.

Comment (blibliboe) [reply src/data/get_data.py:23]: I'll approve it as it does work, but I still think that this is not the intended way of resolving the problem

**Review by blibliboe — APPROVED** [review]
Approved

**Pull request merged**